packages/discord.js/src/structures/GuildTemplate.js

```diff
@@ -69,7 +70,7 @@ class GuildTemplate extends Base {
        * The user that created this template
        * @type {User}
        */
-      this.creator = this.client.users._add(data.creator);
+      this.creator = this.creator ? this.creator._patch(data.creator) : new User(this.client, data.creator);
```

Comment:
If you make `User#_patch` return something, we'll be able to clean this up:

```suggestion
      this.creator = this.creator?._patch(data.creator) ?? new User(this.client, data.creator);
```